index.html

```diff
@@ -29,38 +29,35 @@
   <meta property="twitter:description" content="Mapping a variety of local 350 groups, events and climate actions around the world.">
   <meta property="twitter:image" content="http://350.org/sites/all/themes/threefifty/logo.gif">
 
-  <link rel="stylesheet" href="css/leaflet.css" />
+  <link rel="stylesheet" href="bower_components/leaflet/dist/leaflet.css" />
   <!--[if lte IE 8]><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.ie.css" /><![endif]-->
   <!--[if IE]><style type="text/css">.leaflet-top { top: 60px; }</style><![endif]-->
   <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.js"></script>
 
   <!-- Javascript -->
-  <script type="text/javascript" src="js/jquery-1.9.1.min.js"></script>
+  <script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js"></script>
 
-  <script type="text/javascript" src="markercluster/leaflet.markercluster.js"></script>
-  <link rel="stylesheet" href="markercluster/MarkerCluster.css" />
-  <link rel="stylesheet" href="markercluster/MarkerCluster.Default.css" />
-  <!--[if lte IE 8]><link rel="stylesheet" href="markercluster/MarkerCluster.Default.ie.css" /><![endif]-->
+  <script type="text/javascript" src="bower_components/leaflet.markercluster/dist/leaflet.markercluster.js"></script>
+  <link rel="stylesheet" href="bower_components/leaflet.markercluster/dist/MarkerCluster.css" />
+  <link rel="stylesheet" href="bower_components/leaflet.markercluster/dist/MarkerCluster.Default.css" />
 
   <script src="https://cdn.jsdelivr.net/leaflet.esri/1.0.3/esri-leaflet.js"></script>

```

Comment:
Is there a reason to keep esri-leaflet.js pointed at the third-party CDN instead of using the one installed by bower?